src/config.js

```diff
@@ -41,6 +50,11 @@ export class BotConfig {
     // Response when bot tries to post the exact same response again
     duplicateResponseText = "Please read my previous message...";
 
+    // Checks if the bot is currently muted
+    get isMuted() {
```

Comment:
maybe make it a function to unify with how how we name `is*` methods? That's on me for later cleanup, looks good